R/occ_issues_lookup.r

```diff
@@ -1,17 +1,59 @@
 #' Lookup occurrence issue definitions and short codes
 #'
 #' @export
-#' @param issue Full name of issue, e.g, CONTINENT_COUNTRY_MISMATCH
-#' @param code an issue short code, e.g. ccm
+#' @param issue Full name of occurrence issue, e.g, CONTINENT_COUNTRY_MISMATCH
+#' @param code an occurrence issue short code, e.g. ccm
 #' @examples
 #' occ_issues_lookup(issue = 'CONTINENT_COUNTRY_MISMATCH')
 #' occ_issues_lookup(issue = 'MULTIMEDIA_DATE_INVALID')
 #' occ_issues_lookup(issue = 'ZERO_COORDINATE')
 #' occ_issues_lookup(code = 'cdiv')
 
 occ_issues_lookup <- function(issue=NULL, code=NULL){
-  if(is.null(code))
+
```

Comment:
As you merged the gbifissues table as a single table, the functionality here `occ_issues_lookup` and `name_issues_lookup` are very similar. I would [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this and abstract away the issue lookup as a single function (`gbif_issues_lookup`)

Reply:
Thanks @stijnvanhoey : nice idea.

I did it just because the function `occ_issues_lookup` already exists in `rgbif` in file [`occ_issues_lookup.r`](https://github.com/ropensci/rgbif/blob/master/R/occ_issues_lookup.r) and it matches nice with `occ_issues()` in [`occ_issues.r`](https://github.com/ropensci/rgbif/blob/master/R/occ_issues.r). As @sckott and I decided to have `name_issues` as counterpart of `occ_issues`, I thought to maintain the difference between _name-_ and _occ-_ functions also the `*_lookup` one. I tried to limit as much as possible the repetitions by using the common function `check_issues` in `zzz.r`. 

However, `gbif_issues_lookup` would form a nice pair with `gbif_issues`: the first returning the details of the issue (type: `name` or `occurrence` included) the last one returning all table.

So, a question for @sckott : how would you like to have this implemented? I vote for @stijnvanhoey's idea.

Reply:
Merging seems like a good idea

Reply:
Finally I will work on this now: this is the only issue which avoids to push the PR a step further. Other comments are general issues/discussions.. Sorry for months delay.

Reply:
See a35a1bfa2862861099f3b47e0b28a95f81fa4121